go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/xenowits/nakamoto-coefficient-calculator
 
-go 1.17
```

Comment:
thanks for upgrading it. Hope you have tested it across the project with 1.18

Reply:
I've tested everything including with postgress. The rest of the project works without modifications :)